src/store/reducer.ts

```diff
@@ -1,6 +1,10 @@
 import { combineReducers } from 'redux';
+import { BaseReducer, persistReducer, PersistConfig } from 'redux-persist';
```

Comment:
inconsistent naming

Reply:
Are you talking about this line with the import? If so I unfortunately did not name them, although I can name alias them if there is a more appropriate set of names you think we should use to communicate what we are doing better

Reply:
Oh do you mean because the persistReducer seems like it should be a reducer not a function? I agree

Reply:
yeah that's what I mean sorry